cluster/cluster.go

```diff
@@ -27,6 +27,7 @@ type Cluster interface {
 	// Get all of the services (optionally, from a specific namespace), excluding those
 	AllControllers(maybeNamespace string) ([]Controller, error)
 	SomeControllers([]flux.ResourceID) ([]Controller, error)
+	IsAllowedResource(flux.ResourceID) bool
```

Comment:
It's kind of a shame to have a _whoooole_ new method in the interface for this pretty small thing .. could the namespace guard be contained in just the implementations (`kubernetes.Cluster` and `kubernetes.Manifest`)?

Reply:
Problem is that (at least as is) it needs to be accessed by both the daemon and `kubernetes.Cluster`. The later is needed to filter namespaces when accessing the policies.

Reply:
(I'm shelving this for the minute, since I couldn't come up with better code inside a modest timebox)